### DISCUSSION TITLE
How to change column numbers?

### DISCUSSION THREAD

**Discussion by sandsoncosta** [hbstack/.github#79] [Q&A] [answered]
I've searched for ways to change the number of columns on blog and I haven't been able to.
I don't know if I searched correctly. I'm new to this.
Actually:
![image](https://github.com/user-attachments/assets/5edae2c4-538a-4487-8fd0-12d8d03f047f)

I would like:
![image](https://github.com/user-attachments/assets/1677128c-41df-4db6-8112-64f68519cbcc)

**Comment by razonyang** [comment, marked as answer]
Tweak the `hb.blog.list_cols_md` and `hb.blog.list_cols_lg` parameters, read more on [Blog's site parameters](https://hbstack.dev/modules/blog/overview/#site-parameters).

**Reply by sandsoncosta** [reply]
Thank you very much!
